config/openhpc.yml

```diff
@@ -108,9 +122,11 @@ gluster_volume_name: openhpc_vol
 gluster_src: "localhost:/{{ gluster_volume_name }}"
 gluster_mnt: /mnt/gluster
 gluster_converged_disks: true
+gluster_block_devices:
+- md0
 
 # Admins who can login as centos user
-cluster_users:
+cluster_admins:
```

Comment:
Are these actually admins? Don't they just get user accounts on the login node?

Reply:
Yes, distinction is that users don't have root privileges, admins do.